boards/arm64/xenvm/Kconfig.defconfig

```diff
@@ -15,4 +15,10 @@ config XEN_INITIAL_DOMAIN
 config BOARD
 	default "xenvm"
 
+config XENVM_USE_GIC_V3
+	bool "Xen VM using GICv3"
+	default n
```

Comment:
not needed, this defaults already to `n`

Reply:
ok, will fix